app/src/main/java/de/westnordost/streetcomplete/quests/max_speed/AddMaxSpeedForm.kt

```diff
@@ -121,6 +127,32 @@ class AddMaxSpeedForm : AbstractQuestFormAnswerFragment<MaxSpeedAnswer>() {
         else -> null
     }
 
+    private fun enableAppropriateLabelsForSlowZone(layoutWithSign: FrameLayout) {
+        val position = countryInfo.slowZoneLabelPosition
+        val text = countryInfo.slowZoneLabelText
+
+        // no label
+        if (position == null && text == null) {
```

Comment:
Please remove these strict checks from the code. This is just the UI, I'd rather have the zone sign not display 100% correctly than crash the app if data is not as expected. E.g. if position is not as expected, just use "top". If text is null, just leave the label visibility on GONE.

```kotlin
val label: MaterialTextView = layoutWithSign.findViewById(
    if (position == "bottom") R.id.slowZoneLabelBottom else R.id.slowZoneLabelTop
)
label.visibility.isGone = countryInfo.slowZoneLabelText == null
label.text = countryInfo.slowZoneLabelTex
```

Reply:
Removed.